k9mail/src/main/java/com/fsck/k9/activity/compose/ComposeCryptoStatus.java

```diff
@@ -102,6 +103,9 @@ public boolean isSigningEnabled() {
         return cryptoMode != CryptoMode.DISABLE && signingKeyId != null;
     }
 
+    public boolean isCompatModeEnabled() {
+        return enableCompat;
+    }
 
```

Comment:
ComposeCryptoStatus is somewhat technique agnostic, where as this is very much an OpenPGP setting. Just wondering whether you've considered SMIME while developing this?